conan/tools/microsoft/msbuild.py

```diff
@@ -13,7 +13,7 @@ def msbuild_verbosity_cmd_line_arg(conanfile):
             "quiet": "Quiet",
             "verbose": "Detailed",
         }.get(verbosity)
-        return f'/verbosity:{verbosity}'
+        return f'-verbosity:{verbosity}'
```

Comment:
It seems that ``-`` is supported long time ago, apparently at least VS 2013, so this should be safe